redaxo/src/addons/cronjob/bin/run

```diff
@@ -1,24 +1,9 @@
 #!/usr/bin/env php
 <?php
-
-if (PHP_SAPI !== 'cli') {
-    echo 'error: this script may only be run from CLI';
-    exit(10);
-}
-
-set_time_limit(0);
+// old cronjob-run script, just kept for backwards compatibility reasons.
+// you should invoke the redaxo console with "cronjob:run" directly instead.
 
 // get "redaxo" dir
 // do not use __DIR__ to  be compatible to lower php versions until version check in boot.php
 $dir = dirname(dirname(dirname(dirname(dirname(__FILE__)))));
-chdir($dir);
-
-$REX['REDAXO'] = true;
-$REX['HTDOCS_PATH'] = '../';
-$REX['BACKEND_FOLDER'] = basename($dir);
-
-define('REX_CRONJOB_SCRIPT', true);
-
-require $dir . '/src/core/boot.php';
-
-require rex_path::core('packages.php');
+passthru('php '. $dir .'/bin/console cronjob:run');
```

Comment:
muss/kann man das php commando an dieser stelle allgemeiner bekommen, statt hier darauf zu vertrauen dass es im PATH sein muss?

Reply:
Man könnte alternativ eventuell einen ganz anderen Weg nehmen, `$_SERVER['argv']` manipulieren und dann die console per include einbinden. Weiß aber nicht, ob das wirklich funktioniert.

Reply:
Ansonsten: in symfony/process ist ein PhpExecutableFinder drin: https://github.com/symfony/process/blob/master/PhpExecutableFinder.php
Aber symfony/process haben wir bei uns nicht drin.

Aber fände besser, wenn wir das Problem ganz umgehen würden.

Reply:
wir können die datei auch so lasswn wie sie war und nur die docs anpassen, sowie nen deprecated kommentar reinmachen

Reply:
Was sagst du denn zu meiner Idee oben mit include?

Reply:
Können wir auch testen. Da das alles eher workarounds sind, würde ich vermutlich einfach alles so lassen und nur als deprecated dokumentieren

Reply:
Wobei du dann auch die boot.php wieder anpasen musst, statt generell auf PHP_SAPI zu prüfen, speziell auf `rex::getConsole()`.
Aber ist für mich auch absolut ok, und vielleicht insgesamt das beste.